packages/theming/theme-types/src/Color.types.ts

```diff
@@ -19,7 +19,9 @@ export type ThemeColorDefinition = Palette & {
   bodyText: ColorValue;
   subText: ColorValue;
   disabledText: ColorValue;
-  [key: string]: ColorValue;
+
+  // There is this alias value existing to take custom color names.
```

Comment:
Sorry, this comment doesn't make much sense to me... could you try explaining what you're trying to convey?